### PR TITLE
代码优化，删除多余无用的代码， 解决 #7653

### PR DESCRIPTION
**Pull request by caomingjie-code**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change
我在看源码的时候发现无用多余的代码，并提交了 Issues #7653  ,现在提交一个pr，删除无用的代码
优化多余代码  #7653

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: api/src/main/java/com/alibaba/nacos/api/remote/PayloadRegistry.java [hunk 1]
```diff
@@ -79,9 +79,6 @@ static void register(String type, Class<?> clazz) {
         if (Modifier.isAbstract(clazz.getModifiers())) {
             return;
         }
-        if (Modifier.isInterface(clazz.getModifiers())) {
```
Comment (li-xiao-shuang) [inline api/src/main/java/com/alibaba/nacos/api/remote/PayloadRegistry.java:82]: 这里不能删除吧  如果是修饰符 是interface 不就继续执行了？

Comment (caomingjie-code) [reply api/src/main/java/com/alibaba/nacos/api/remote/PayloadRegistry.java:82]: > 这里不能删除吧 如果是修饰符 是interface 不就继续执行了？

接口默认属于抽象。但是类有两种，抽象的和普通的（非抽象的）

Comment (caomingjie-code) [reply api/src/main/java/com/alibaba/nacos/api/remote/PayloadRegistry.java:82]: 就比如说，interface中的方法，及时你没有写abstract。 默认也是 abstract。 interface的本质就是抽象。让其实现内部的动作行为，这是java语言在设计interface时的初衷，随着jdk版本的升级，后期又加入了的default 。

**Comment by KomachiSion** [comment]
duplicate with #7839

**Comment by caomingjie-code** [comment]
> duplicate with #7839

这哥们儿挺会捡漏啊。影响开源的兴趣。

**Pull request closed without merging**